docs: add docs for verify_new_address and notify_previous_addresses (#2597)
ory/docs: docs/kratos/hooks/01_configure-hooks.mdx

| [`session`](../../actions/session.mdx)                                     | Signs in the user immediately after they create an account.                                                  | For use only with the `after` registration flow. To use it, you must define secrets for secret rotation.                                                                                                        |
| [`revoke_active_sessions`](../../actions/revoke-active-sessions.mdx)       | Revokes all other active sessions of the user on successful login.                                           | For use only with the login flow.                                                                                                                                                                               |
| [`require_verified_address`](../../actions/require-verified-address.mdx)   | Allows users to sign in only when they verified their email address.                                         | For use only with the `after` login flow, `password` method only.                                                                                                                                               |

> | [`verify_new_address`](../../actions/verify-new-address.mdx)               | Defers an address change until the user verifies the new address.                                            | For use only with the `after` settings flow, `profile` method only.                                                                                                                                             |

Reviewer: ```suggestion
| [`verify_new_address`](../../actions/verify-new-address.mdx)               | Changes the user's address after they verify the new address.                                            | For use only with the `after` settings flow, `profile` method only.                                                                                                                                             |
```
